packages/react-components/src/auth-test-utils.tsx

```diff
@@ -40,7 +40,9 @@ export const Auth0Provider: React.FC<{
         loading: !auth0Client,
         popupOpen: false,
         isAuthenticated: false,
-        getIdTokenClaims: notImplemented('getIdTokenClaims'),
+        getIdTokenClaims: auth0Client
+          ? (...args) => auth0Client.getIdTokenClaims(...args).then()
```

Comment:
`.then()`?

Reply:
I would suggest you hit `cmd + g` next time 😉